setup.py

```diff
@@ -2,28 +2,30 @@
 import os
 from glob import glob
 
-package_name = 'eeg_publisher'
+package_name = "eeg_publisher"
 
 setup(
     name=package_name,
-    version='0.0.0',
-    packages=find_packages(exclude=['test']),
+    version="0.0.0",
+    packages=find_packages(exclude=["test"]),
     data_files=[
-        ('share/ament_index/resource_index/packages',
-            ['resource/' + package_name]),
-        ('share/' + package_name, ['package.xml']),
-        (os.path.join('share', package_name, 'launch'), glob('launch/*.launch.*') + glob('launch/*.py')),
+        ("share/ament_index/resource_index/packages", ["resource/" + package_name]),
+        ("share/" + package_name, ["package.xml"]),
+        (
+            os.path.join("share", package_name, "launch"),
+            glob("launch/*.launch.*") + glob("launch/*.py"),
+        ),
     ],
-    install_requires=['setuptools'],
+    install_requires=["setuptools"],
     zip_safe=True,
-    maintainer='Ahmed Al-Hindawi',
-    maintainer_email='a.al-hindawi@ucl.ac.uk',
-    description='TODO: Package description',
-    license='TODO: License declaration',
-    tests_require=['pytest'],
+    maintainer="Ahmed Al-Hindawi",
+    maintainer_email="a.al-hindawi@ucl.ac.uk",
+    description="TODO: Package description",
+    license="TODO: License declaration",
```

Comment:
Sorry, I think I missed this bits on the previous PR. Could you please add a brieft description and add licence which we have "Apache License 2.0"?